src/dev-app/BUILD.bazel

```diff
@@ -133,8 +133,8 @@ ts_devserver(
         "@npm//:node_modules/moment/min/moment-with-locales.min.js",
         "@npm//:node_modules/rxjs/bundles/rxjs.umd.min.js",
         "@npm//:node_modules/systemjs/dist/system.js",
+        "@npm//:node_modules/tslib/tslib.js",
```

Comment:
Was `tslib` missing previously? or was it provided by `@npm//material-components-web`

Reply:
Yeah transitively brought in by `material-components-web` (which was by accident).